code-for-canada/project-thundercat
# [Inbox] Use pre-wrap css instead of making paragraphs

# Description

Some of our components recognized new lines in emails and others didn't. Some created new paragraphs for each new line.

This makes everything consistent (email body, response body, tasks, reasons for action) with css whitespace. This updates within the inbox and the instructions.

This CSS is compatible with IE and removes the need for the unit test about paragraphs.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

<img width="1285" alt="Screen Shot 2019-04-29 at 10 51 13 AM" src="https://user-images.githubusercontent.com/4640747/56904738-e4412e80-6a6c-11e9-8cfe-42b8107352f5.png">
<img width="726" alt="Screen Shot 2019-04-29 at 10 51 59 AM" src="https://user-images.githubusercontent.com/4640747/56904739-e4412e80-6a6c-11e9-9843-649288022b90.png">
<img width="739" alt="Screen Shot 2019-04-29 at 10 52 11 AM" src="https://user-images.githubusercontent.com/4640747/56904740-e4d9c500-6a6c-11e9-85e1-2fd3003c01b4.png">


# Testing

Manual steps to reproduce this functionality:

1.  Go to sample test.
2.  Click test examples. Notice email formatting.
3. Click inbox. notice email formatting.
4. Add a response with new line formatting and notice formatting.

# Checklist

Applicable for all code changes.
